docs/source/assets/styles/layout/_navbar.sass

```
@@ -1,3 +1,13 @@
+// FiftyOne Labs icon in sidebar nav
+.nav.bd-sidenav a.reference.nav-labs::after
```

Stylelint (17.9.0)

[error] Unknown word a.reference.nav-labs (CssSyntaxError)

(CssSyntaxError)